elasticsearch/scripts/createAlias.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# To create a index in elasticsearch
+#
+
+# shards & replicas are default values.
+#
+
+ALIAS=$1
+INDEX=$2
+
+if [ ! -n "${ALIAS}" ] ; then
+    echo -e $RED "Devi passarmi il nome dell'Alias" $Z;
+    exit 1;
+fi
+if [ ! -n "${INDEX}" ] ; then
+    echo -e $RED "Devi passarmi il nome dell'Indice" $Z;
+    exit 1;
+fi
+
+generate_create_msg() {
+  cat <<EOF
+{
+    "actions" : [
+      { "add" : { "index" : "$INDEX", "alias" : "$ALIAS" } }
+    ]
+}
+EOF
+}
+
+curl -u elastic:elastic -X POST "http://elasticsearch:9200/_aliases" -H 'Content-Type: application/json' -d"$(generate_update_msg)"
```

Comment:
can we load the server url from an environment variable? (here and in all other scripts)